layouts/partials/head.meta.html

```diff
@@ -36,7 +36,26 @@
 
 <link rel="sitemap" type="application/xml" href="/sitemap.xml"/>
 
-{{- if $DATA.algolia -}}
+<!-- useCoveo := false will reenable Algolia search -->
+{{- $useCoveo := true -}}
+{{- if $useCoveo -}}
```

Comment:
Can we have this reference the new `config.toml` values? If there's a coveo section defined, use it, otherwise check for algolia

Reply:
thanksss, forgot about this part, done! 
https://github.com/lukeed/cloudflare-docs/pull/11/commits/26d1c79940ac5045d6e21b622ebfd8345d26d0d3